source/features/github-bugs.css

```diff
@@ -58,6 +58,11 @@
 	margin-right: 4px !important;
 }
 
+/* Restore zebra-colored rows in CSV files */
+.blob-wrapper.type-csv table tr:nth-child(2n) {
+	background-color: var(--color-canvas-subtle);
+}
```

Comment:
I'm not seeing this in the latest version, so I'll drop it (also because unrelated). Feel free to open a new PR

<img width="831" alt="Screen Shot 4" src="https://user-images.githubusercontent.com/1402241/197964736-186f01c7-61dc-45d3-9426-ee28b87831b6.png">